master/internal/trial.go

```diff
@@ -105,12 +106,23 @@ type (
 		socket      *websocket.Conn
 	}
 
-	// trialWatchPreemption begins watching if the trial has been preempted.
+	// trialWatchPreemptionReq begins watching if the trial has been preempted.
 	// The trial responds to this message with a channel of bools, where sends of true
 	// indicate to preempt and sends of false are used to synchronize (e.g. you want to
 	// block until you receive _something_ but not until the first preemption).
-	trialWatchPreemption   struct{ id uuid.UUID }
-	trialUnwatchPreemption struct{ id uuid.UUID }
+	trialWatchPreemptionReq  struct{ id uuid.UUID }
```

Comment:
Can you name it to be `trialWatchPreemption`?

```suggestion
	trialWatchPreemption  struct{ id uuid.UUID }
```